app/models/spree/taxjar.rb

```diff
@@ -40,12 +40,32 @@ def calculate_tax_for_order
 
     private
 
+      def tax_address_country
```

Comment:
@pikender - Lets name it to tax_address_country_iso to avoid any confusion. Same for tax_address_state.